java-script/package.json

```diff
@@ -0,0 +1,45 @@
+{
+    "name": "creditcoin-ts",
+    "version": "0.0.1",
+    "description": "Polkadot JS examples and integration tests for Creditcoin Substrate",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/gluwa/creditcoin-substrate.git"
+    },
+    "license": "Unlicense",
+    "bugs": {
+        "url": "https://github.com/gluwa/creditcoin-substrate/issues"
+    },
+    "homepage": "https://github.com/gluwa/creditcoin-substrate#readme",
+    "packageManager": "yarn@3.2.0",
+    "scripts": {
+        "start": "npx ts-node -O '{\"module\":\"commonjs\"}' ./src/index.ts",
```

Comment:
doesn't need `npx` at the beginning.

Reply:
Looks like it does:

```
yarn start
yarn run v1.22.17
$ ts-node -O '{"module":"commonjs"}' ./src/index.ts
/bin/sh: 1: ts-node: not found
error Command failed with exit code 127.
```

Reply:
weird, worked for me without it

Reply:
works for me now too. I must have done something wrong in between. Updated.